ui/src/data/source.tsx

```diff
@@ -970,3 +1035,86 @@ function normalizeRequestedAttributes(attributes: string[]) {
     ),
   ];
 }
+
+function fromAPICohort(cohort: tanagra.CohortV2): tanagra.Cohort {
```

Comment:
Do you plan to keep this translation between the API and UI schemas long-term, or is it just a short-term bridge to keeping the existing code working? On the backend, I separate the API serialization from the internal objects, just wondering if you do the same thing for UI code.